orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/UpdateLaunchConfigTask.groovy

```diff
@@ -68,6 +68,7 @@ class UpdateLaunchConfigTask implements Task, DeploymentDetailsAware, CloudProvi
   private getAwsOps(Stage stage) {
     def operation = new HashMap(stage.context)
     operation.amiName = getImage(stage)
+    operation.asgName = operation.asgName ?: operation.serverGroupName
```

Comment:
Unfortunately, it's required by Clouddriver in the `updateAsgLaunchConfiguration` operation, and I didn't want to break anybody already using it.